### ISSUE TITLE
Training loss uses MSE while validation loss uses L2

### ISSUE THREAD

**Issue by shervinsahba** [Jan-Williams/pyshred#2]
Training uses MSELoss while validation uses L2 loss, where L2 = sqrt(N*MSE). It doesn't affect the model much, but the loss types should match for tuning and comparative purposes. You'd probably want to change one or the other for consistency.

Training MSELoss
https://github.com/Jan-Williams/pyshred/blob/3a35e9fe6c3e27f30af0878b80cc49f3329c4ba2/models.py#L78-L89

Validation L2 loss
https://github.com/Jan-Williams/pyshred/blob/3a35e9fe6c3e27f30af0878b80cc49f3329c4ba2/models.py#L96-L98
